src/libpcp_web/src/webgroup.c

```diff
@@ -1867,6 +1868,39 @@ webgroup_scrape(pmWebGroupSettings *settings, context_t *cp,
     return sts < 0 ? sts : 0;
 }
 
+typedef struct webscrape {
+    pmWebGroupSettings	*settings;
+    struct context	*context;
+    sds			*msg;
+    int			status;
+    unsigned int	numnames;	/* current count of metric names */
+    sds			*names;		/* metric names for batched up scrape */
+    struct metric	**mplist;
+    pmID		*pmidlist;
+    int			numfilters;	/* current count of filters */
+    sds			*filters;
+    enum matches	match;		/* type of matching to use for filters */
+    regex_t		*regex;		/* for filter compiled regular expressions */
+    void		*arg;
+} webscrape_t;
+
+static int
+filtered(struct webscrape *filter_info, const char *name)
+{
+    int i;
+    for (i = 0; i<filter_info->numfilters; ++i) {
+	if (filter_info->match == MATCH_EXACT && strcmp(filter_info->filters[i], name) == 0)
+	    return 1;
+	if (filter_info->match == MATCH_REGEX &&
+	    regexec(&(filter_info->regex[i]), name, 0, NULL, 0) == 0)
+	    return 1;
+	if (filter_info->match == MATCH_GLOB &&
+	    fnmatch(filter_info->filters[i], name, 0) == 0)
+	    return 1;
```

Comment:
Small code consistency thing - can we make the first "if" branch line-splitting match the other two cases here?  (i.e. insert EOL after "&&").